docs/DeveloperGuide.md

```diff
@@ -324,13 +329,149 @@ Below are instructions to perform manual testing of the application. Please refe
 2. Test Case: `add mod CS2113T 4` (Continuation from Test Case 1) <br>
    Expected: No new module is added. Error details in the message shows that a module of the same name already exists.
 3. Test Case: `add mod CS2101` <br>
-   Expected: No new module is added. Error details in the message shows that there are invalid compulsory parameters in the command.
+   Expected: No new module is added. Error details in the message shows that there are missing modular credits.
+
+<br>
+
+### Adding a task
+1. Prerequisite: There are existing modules in the application.
+2. Assumption: You have a module `CS2113T` added.
+3. Test Case: `add task "start PE" -m CS2113T` <br>
+   Expected: A task with name `start PE` is added under the module CS2113T.
+4. Test Case: `add task Invalid Name` <br>
+   Expected: No task is added. Error details in the message shows that the task name is invalid.
+
 <br>
 
 ### Deleting a module
-1. Prerequisite: There are existing modules in the application, after testing Adding a module.
-2. Test Case: `del mod CS2113T` <br>
+1. Prerequisite: There are existing modules in the application.
+2. Assumption: You have a module `CS2113T` added, but not `CS2101`.
+3. Note: If you have added tasks to a module, deleting that module will prompt a confirmation to delete. Typing `yes` will delete that module. 
+4. Test Case: `del mod CS2113T` <br>
    Expected: The module CS2113T is deleted.
-3. Test Case: `del mod CS2101` <br>
-   Expected: If CS2101 does not exist, no module is deleted. Error details in the message shows that there are no such module. 
+5. Test Case: `del mod CS2101` <br>
+   Expected: No module is deleted. Error details in the message shows that there are no such module. 
+
+<br>
+
+### Deleting a task
+1. Prerequisite: There are existing tasks in the application.
+2. Assumption: You have the module `CS2113T` added with at least one task added. 
+3. Test Case: `del task 1 -m CS2113T` <br>
+   Expected: If there exists a module CS2113T, and it has at least one task, the first task will be deleted.
```

Comment:
I think you can omit the `If there exists a module CS2113T, and it has at least one task` since it is already mentioned in the assumption.

Reply:
yup removed